Automattic/wp-calypso
# Developer UX: remove feature flag and deploy SMP submenu

- Closes https://github.com/Automattic/dotcom-forge/issues/1852

## Proposed Changes

* Remove the feature flag `dev/developer-ux` and make the SMP submenu visible to all users.

## Testing Instructions



* Go to `/sites`
* Click on the actions menu on any site
* Observe the Hosting Configuration is visible and it show the correct content. Upsell for free sites and Submenu items for Business+.

## Pre-merge Checklist
